src/com/lftechnology/java/training/alina/basic/BasicConstants.java

```diff
@@ -0,0 +1,18 @@
+package com.lftechnology.java.training.alina.basic;
+
+/**
+ * Collected basic constants to be used
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ *
+ */
+public final class BasicConstants {
+	public static final String SUNDAY = "Sunday";
+	public static final String MONDAY = "Monday";
+	public static final String TUESDAY = "Tuesday";
+	public static final String WEDNESDAY = "Wednesday";
+	public static final String THURSDAY = "Thursday";
+	public static final String FRIDAY = "Friday";
+	public static final String SATURDAY = "Saturday";
+	public static final String INVALID_DAY = "Not a valid day";
+}
```

Comment:
:+1:  for using Constants. Once suggestion we use name CommonConstants instead of BasicConstants